.circleci/config.yml

```diff
@@ -41,7 +41,7 @@ jobs:
         command: |
           rustc --version
           cargo --version
-          cargo test --all --all-features
+          cargo test --all --features "default softsign yubihsm yubihsm-mock"
```

Comment:
Why is this change necessary?

Reply:
The test harness does not work with the Ledger since I haven't implemented a ledger-mock.